### PR TITLE
Fix filtering

### PR DESCRIPTION
**Pull request by brendin-nye**
### Context

Filtering was implemented in pull request #11. In its current state, there is a small import error in filtering.go. Filtering implementation appears to assume JSON(B) for the `data` columns in `notes` and `occurrences` tables.

### Description

This PR brings the filtering implementation into a working state by changing the import of `expr` to the one implemented in grafeas repo. `data` columns are changed to JSONB and database operation code in pgsqlstore.go is updated accordingly. Some dependency updates and fix some deprecation warnings with `ptypes timestamp.Now()`.

### CODE REVIEW

**Comment by brendin-nye** [comment]
Woops, something went wrong with signing CLA, I'll get that fixed.

File: go/v1beta1/main/main.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package main
```
Comment (brendin-nye) [inline go/v1beta1/main/main.go:1]: This is for use in a future PR where code in this repo will be built and run in the Dockerfile.

File: go/v1beta1/storage/queries.go [hunk 2]
```diff
@@ -24,23 +24,16 @@ const (
...
 		CREATE TABLE IF NOT EXISTS occurrences (
 			id SERIAL PRIMARY KEY,
 			project_name TEXT NOT NULL,
 			occurrence_name TEXT NOT NULL,
-			data TEXT,
+			data JSONB,
 			note_id int REFERENCES notes NOT NULL,
 			UNIQUE (project_name, occurrence_name)
-		);
-		CREATE TABLE IF NOT EXISTS operations (
```
Comment (brendin-nye) [inline go/v1beta1/storage/queries.go:38]: `operations` table didn't appear to be in use anywhere.

**Comment by aethanol** [comment]
cc @wkozlik @nyc ❤️

**Review by wkozlik — APPROVED** [review]
LGTM. Thank you for your contributions.

**Pull request merged**